client/signup/steps/design-picker/index.jsx

```diff
@@ -174,6 +179,21 @@ export default function DesignPickerStep( props ) {
 						align="left"
 					/>
 				}
+				categoriesFooter={
+					useBlankCanvasButton && (
+						<div className="design-picker__categories-footer">
+							<Button
+								className="design-picker__categories-footer-button"
+								isSecondary
+								onClick={ () =>
+									pickDesign( designs.find( ( design ) => isBlankCanvasDesign( design ) ) )
+								}
+							>
+								{ translate( 'Use Blank Canvas' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Blank Canvas' )` *ES Score: 8*